packages/site-preset-styles/package.json

```diff
@@ -35,8 +35,7 @@
     "@jpmorganchase/mosaic-content-editor-plugin": "0.1.0-beta.34",
     "@jpmorganchase/mosaic-site-components": "0.1.0-beta.34",
     "@jpmorganchase/mosaic-layouts": "0.1.0-beta.34",
-    "@jpmorganchase/mosaic-theme": "0.1.0-beta.34",
-    "prismjs": "^1.23.0"
+    "@jpmorganchase/mosaic-theme": "0.1.0-beta.34"
```

Comment:
does this not export Prism CSS from index.js

import 'prismjs/themes/prism.css';

Reply:
preset styles did...been removed https://github.com/jpmorganchase/mosaic/pull/375/files#diff-689daf31a0e205fe397769cedbdb31a32f0c9d42088fed8c77d25457e3591c5a